examples/tenant_tutorial/tenant_tutorial/urls_public.py

```diff
@@ -1,6 +1,6 @@
-from django.conf.urls import patterns, url
```

Comment:
I don't think this is backward compatible, maybe add a django version flag so that it works with django 1.10 as well earlier versions too.